app/assets/src/components/views/compare/SamplesHeatmapVis.jsx

```diff
@@ -0,0 +1,155 @@
+import DataTooltip from "../../ui/containers/DataTooltip";
+import Heatmap from "../../visualizations/heatmap/Heatmap";
+import PropTypes from "prop-types";
+import React from "react";
+import { openUrl } from "../../utils/links";
+import SvgSaver from "svgsaver";
+
+class SamplesHeatmapVis extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      nodeHover: null
+    };
+
+    this.heatmap = null;
+    this.scale = this.props.scale;
+
+    this.handleNodeHover = this.handleNodeHover.bind(this);
+
+    // TODO: yet another metric name conversion to remove
+    this.metrics = [
+      { key: "NT.aggregatescore", label: "Score" },
+      { key: "NT.zscore", label: "NT Z Score" },
+      { key: "NT.rpm", label: "NT rPM" },
+      { key: "NT.r", label: "NT r (total reads)" },
+      { key: "NR.zscore", label: "NR Z Score" },
+      { key: "NR.rpm", label: "NR rPM" },
+      { key: "NR.r", label: "NR r (total reads)" }
+    ];
+
+    this.handleCellClick = this.handleCellClick.bind(this);
```

Comment:
Use arrow function

Reply:
Done
A note (I OCD about this... :) ): i'm not a fan of mixing both. Should we start enforcing only arrow function, event if this is not used?